app/code/Magento/Checkout/Test/Mftf/Test/StorefrontVerifyGuestCartMinimumAmountTest.xml

```diff
@@ -0,0 +1,68 @@
+            <createData entity="DefaultTaxRuleWithCustomTaxRate" stepKey="createTaxRuleUSNY">
+                <requiredEntity createDataKey="createTaxRateUSNY" />
+            </createData>
+            <createData entity="ApiCategory" stepKey="createCategory"/>
+            <createData entity="defaultSimpleProduct" stepKey="simpleProduct">
+                <requiredEntity createDataKey="createCategory"/>
+                <field key="price">93.00</field>
+            </createData>
+            <actionGroup ref="CliCacheCleanActionGroup" stepKey="cleanInvalidatedCaches">
+                <argument name="tags" value="config full_page"/>
+            </actionGroup>
+        </before>
+        <after>
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+            <deleteData createDataKey="simpleProduct" stepKey="deleteProduct"/>
+            <deleteData createDataKey="createTaxRuleUSNY" stepKey="deleteTaxRuleUSNY"/>
+            <deleteData createDataKey="createTaxRateUSNY" stepKey="deleteTaxRateUSNY"/>
+            <createData entity="DefaultShippingMethodsConfig" stepKey="defaultShippingMethodsConfig"/>
+            <createData entity="DefaultMinimumOrderAmount" stepKey="defaultMinimumOrderAmount"/>
+        </after>
+        <actionGroup ref="AssertProductNameAndSkuInStorefrontProductPageByCustomAttributeUrlKeyActionGroup" stepKey="openProductPageAndVerifyProduct">
+            <argument name="product" value="$simpleProduct$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddProductToCartWithQtyActionGroup" stepKey="addSimpleProductToTheCart">
+            <argument name="productQty" value="1"/>
+        </actionGroup>
+        <actionGroup ref="ClickViewAndEditCartFromMiniCartActionGroup" stepKey="clickMiniCart"/>
+        <waitForElementVisible selector="{{CheckoutCartSummarySection.proceedToCheckoutDisabled}}" stepKey="goToCheckoutDisabled"/>
+        <actionGroup ref="AssertMessageCustomerChangeAccountInfoActionGroup" stepKey="assertMinimumAmountOrderMessage">
+            <argument name="message" value="Minimum order amount is $100.00"/>
+            <argument name="messageType" value="notice"/>
+        </actionGroup>
+        <actionGroup ref="CheckoutFillEstimateShippingAndTaxActionGroup" stepKey="fillEstimateShippingAndTaxFields">
+            <argument name="address" value="US_Address_NY_Default_Shipping"/>
+        </actionGroup>
+        <click selector="{{CheckoutCartSummarySection.shippingMethodElementId('freeshipping', 'freeshipping')}}" stepKey="selectShippingMethod"/>
+        <scrollTo selector="{{CheckoutCartSummarySection.proceedToCheckout}}" stepKey="scrollToProceedToCheckout" />
+        <actionGroup ref="StorefrontClickProceedToCheckoutActionGroup" stepKey="goToCheckout"/>
+        <comment userInput="Adding the comment to replace waitForPageToLoad action for preserving Backward Compatibility" stepKey="waitForPageToLoad"/>
```

Comment:
Comment block used on refactored mftf tests to preserve backward compatibility and don't needed in new mftf tests

Reply:
Done. Please review again when you have a chance.